.github/stubs/aws/stub-aws-cli.sh

```diff
@@ -83,5 +83,11 @@ if [[ ${1:-} == kms ]] && [[ ${2:-} == sign ]]; then
   exit 0
 fi
 
+if [[ ${1:-} == sns ]] && [[ ${2:-} == publish ]]; then
+  echo "SNS_ARN=some-arn" >> "$GITHUB_ENV"
+  echo "MESSAGE=Message Sent" >> "$GITHUB_ENV"
```

Comment:
```suggestion
  echo "SNS_MESSAGE=Message Sent" >> "$GITHUB_ENV"
```
Prefix with the "stub" service name